client/landing/gutenboarding/get-available-designs.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Internal dependencies
+ */
+const availableDesignsConfig = require( './available-designs-config.json' );
+
+/**
+ * This function reads the available designs from the config. This is written
+ * in CommonJS so that the node script for generating previews can use this to
```

Comment:
The downside is that this is untyped, and currently producing a [TS error](https://app.circleci.com/pipelines/github/Automattic/wp-calypso/64315/workflows/c1d4e7fd-464a-4109-a9e1-61513621f9b0/jobs/780314/steps).

I'd opt for removing the `forceFSE` option ([see](https://github.com/Automattic/wp-calypso/pull/43349#issuecomment-646346637)), which will probably allow us to drop this script (and re-inline the remaining logic into where it was before) -- this should get us rid of the TS error.